circle.yml

```diff
@@ -10,63 +10,25 @@ dependencies:
     - cd ~/curation/data_steward/ && pip install -t lib/ -r requirements.txt 
    
 machine:
+  pre:
+    - ~/curation/data_steward/init_env.sh
+    - cat $HOME/.circlerc
```

Comment:
Does this create .circlerc?

Reply:
No, just shows the contents. It's useful for troubleshooting.